dojo/db_migrations/0201_populate_finding_sla_expiration_date.py

```diff
@@ -0,0 +1,129 @@
+from django.db import migrations
+from django.utils import timezone
+from datetime import datetime
+from django.conf import settings
+from dateutil.relativedelta import relativedelta
+import logging
+
+from dojo.utils import get_work_days
+
+logger = logging.getLogger(__name__)
+
+
+def calculate_sla_expiration_dates(apps, schema_editor):
+    System_Settings = apps.get_model('dojo', 'System_Settings')
+
+    ss, _ = System_Settings.objects.get_or_create()
+    if ss.enable_finding_sla:
```

Comment:
To reduce nesting here, you could simplify to this:

```suggestion
    if not ss.enable_finding_sla:
        return
```